userportaldatamodel/schemas/project_datapoints_schema.py

```diff
@@ -1,14 +1,11 @@
 from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
 from userportaldatamodel.models import ProjectDataPoints
 from marshmallow_sqlalchemy.fields import Nested
-
+from amanuensis.schema import ProjectSchema
 
 class ProjectDataPointsSchema(SQLAlchemyAutoSchema):
-    project = Nested('Project')
+    project = Nested(ProjectSchema)
```

Comment:
You could also do [this](https://github.com/chicagopcdc/userPortalDataModel/blob/1f50f1d70e5b8250712bf27ac309812238bdeec6/userportaldatamodel/schemas/request_schema.py#L11) to avoid the import